src/StateSmith/Output/Sim/HtmlRenderer.cs

```diff
@@ -198,38 +198,54 @@ .console td.timestamp {
         display: none;
       }
 
-      .dropbtn {
+
+
+      /* ----------------------------- Dropdown related start ----------------------------- */
+      
+      .dropdown-button {
         border: none;
         cursor: pointer;
       }
-
-      .dropbtn:hover, .dropbtn:focus {
+      
+      .dropdown-button:hover, .dropdown-button:focus {
```

Comment:
using full english words so that it is hopefully easier for non-english speaking contributors to translate